reflex-dom/java/org/reflexfrp/reflexdom/MainWidget.java

```diff
@@ -87,7 +86,7 @@ public WebResourceResponse shouldInterceptRequest (WebView view, WebResourceRequ
 
         @Override
         public boolean shouldOverrideUrlLoading(WebView view, String url) {
-            if( url != null && !url.startsWith("http://") && !url.startsWith("https://") && !url.startsWith("file://")) {
+            if( url != null && !url.startsWith("http://appassets.androidplatform.net") && !url.startsWith("https://appassets.androidplatform.net") && !url.startsWith("file://")) {
```

Comment:
Shouldn't this bit be included in #452 ?

Pre-existing, but it's odd that above `scheme`/`encodedAuthority` are used but here `startsWith` is used. Maybe we want this de-duplicated